### ISSUE TITLE
1.7.8 Client kicked

### ISSUE THREAD

**Issue by LoXR** [HexagonMC/BungeeCord#194]
### What Hexacord version are you using? Post the output of /bungee
Latest Hexacord [#26](https://ci.lucko.me/job/HexaCord/26/)

### Are you using Bungee plugins? If yes, please list all of them + versionnumber
Yes (BConfReloader) (bungeeAuthMeBridge)(BungeeMSG)(BungeeStaff)(BungeeTablistPlus)(EMPBungee)(FastLogin)(HubMagics)(NuVotifier)(SkinsRestorer)(UltimateFriends)

### How can we reproduce the error/bug? 
Join with a 1.7.8 client

### Is an error message (bungee/spigot/client)? If yes, use pastebin.com and share the link.
No.

### In addidtion to that, please explain your issue as detailed as possible.
Just has a message that pops up "Outdated server! Pretending to be 1.7.8" All other version work fine.

**Comment by spannerman79** [comment]
The oldest you can go is 1.7.10 - so update to **at least** 1.7.10

**Comment by LoXR** [comment]
I can join with 1.7 clients, but cant with 1.7.8? What has updating got to do if all 1.7 clients can join but not for 1.7.8?

**Comment by LoXR** [comment]
Yeah its very odd, i've tried it out on my localhost, and can connect fine.

**Comment by popcorn9499** [comment]
Same plugins or different ones?

**Comment by LoXR** [comment]
Same plugins on both bungee and hub.

**Comment by popcorn9499** [comment]
Have you tried without any plugins to see if it happens then? Because I myself dont have the issue you are experiencing so im trying to figure out whats different and the main one i see is plugins.
